src/AppETH.tsx

```diff
@@ -52,6 +56,19 @@ function AppETH ({ contract, defaultAccount, web3 }: Props): React.ReactElement<
   const [polkadotRecipient, setPolkadotRecipient] = useState(String);
   const [depositAmount, setDepositAmount] = useState(String);
 
+  const [polkaEthBalance, setPolkaEthBalance] = useState(String);
+
+  if(defaultAccount === undefined) {
+    alert("Please install MetaMask to use this application!");
```

Comment:
plz dont use alerts - the div below is fine

Reply:
this is in appeth and app? do we need it in both?

Reply:
Alright I will remove that.